rubygem-foreman_scap_client/foreman_scap_client.spec

```diff
@@ -0,0 +1,90 @@
+%global gem_name foreman_scap_client
+%global config_dir %{_sysconfdir}/%{gem_name}
+
+Name: %{gem_name}
```

Comment:
comps/ has a rubygem- prefix, which is what the packaging guidelines would need too, it should be here?